app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    index_new_directories = command_list[2].rfind("/")
+    new_directories = command_list[2][:index_new_directories]
```

Comment:
Then, when you use this index in the slice, the last symbol gets cut off, 
with command 'mv file.txt file2.txt',
new_directories == a2.tx.